docs/book/src/user/quick-start.md

```diff
@@ -628,3 +629,4 @@ See the [clusterctl] documentation for more detail about clusterctl supported ac
 [provider components]: ../reference/glossary.md#provider-components
 [vSphere getting started guide]: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/
 [workload cluster]: ../reference/glossary.md#workload-cluster
+[legacy-clusterawsadm]: https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/user/quick-start.md#initialization-for-common-providers
```

Comment:
Wouldn't this link to these updated docs once merged?

Reply:
Argh. I meant to copy and paste from a tag.

Reply:
Fixed.